src_c/collisions.c

```diff
@@ -83,15 +83,19 @@ pgIntersection_LineLine(pgLineBase *A, pgLineBase *B, double *X, double *Y,
 static int
 pgCollision_LinePoint(pgLineBase *line, double Cx, double Cy)
 {
```

Comment:
Instead of removing this equation entirely, shall we create an entirely new function with the its original old code?